roles/rke2_common/templates/config.yml.j2

```diff
@@ -0,0 +1,28 @@
+{% if advertise_address is defined and advertise_address %}
+advertise-address: {{ addvertise_adress }}
```

Comment:
seems like a typo to me? {{ addvertise_adress }}

Reply:
thanks @phil-dotchon , fixed in https://github.com/rancherfederal/rke2-ansible/pull/139/commits/8397bf5b57e32b90fb602bac42dad8e6df953a87